connector_prestashop/models/prestashop_backend/common.py

```diff
@@ -0,0 +1,442 @@
+    import_products_since = fields.Datetime('Import Products since')
+    import_refunds_since = fields.Datetime('Import Refunds since')
+    import_suppliers_since = fields.Datetime('Import Suppliers since')
+    language_ids = fields.One2many(
+        comodel_name='prestashop.res.lang',
+        inverse_name='backend_id',
+        string='Languages',
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        index=True,
+        required=True,
+        default=lambda self: self.env['res.company']._company_default_get(
+            'prestashop.backend'),
+        string='Company',
+    )
+    discount_product_id = fields.Many2one(
+        comodel_name='product.product',
+        index=True,
+        required=True,
+        string='Discount Product',
+    )
+    shipping_product_id = fields.Many2one(
+        comodel_name='product.product',
+        index=True,
+        required=True,
+        string='Shipping Product',
+    )
+    importable_order_state_ids = fields.Many2many(
+        comodel_name='sale.order.state',
+        string='Importable sale order states',
+        help="If valued only orders matching these states will be imported.",
+    )
+    active = fields.Boolean(
+        string='Active',
+        default=True
+    )
+    state = fields.Selection(
+        selection='_select_state',
+        string='State',
+        default='draft'
+    )
+
+    verbose = fields.Boolean(help="Output requests details in the logs")
+    debug = fields.Boolean(help="Activate PrestaShop's webservice debug mode")
+
+    matching_product_template = fields.Boolean(string="Match product template")
+
+    matching_product_ch = fields.Selection(
+        [('reference', 'Reference'), ('barcode', 'Barcode')],
+        string="Matching Field for product")
+
+    matching_customer = fields.Boolean(
+        string="Matching Customer",
+        help="The selected fields will be matched to the ref field of the "
+        "partner. Please adapt your datas consequently.")
+    # matching_customer_ch = fields.Many2one(
+    #     comodel_name='prestashop.partner.field', string="Matched field",
+    #     help="Field that will be matched.")
+
+    quantity_field = fields.Selection(
+        [('qty_available', 'Available Quantity'),
+         ('virtual_available', 'Forecast quantity'),
+         ('immediately_usable_qty', 'Available to promise'),
+         ('potential_qty', 'Potential')],
+        help="Some of this options may need some additionnal modules you'll "
+        "have to install by yourself from "
+        "https://github.com/OCA/stock-logistics-warehouse",
+        required=True,
+        default='virtual_available')
+    tz = fields.Selection(
+        _tz_get, 'Timezone', size=64,
+        help="The timezone of the backend. Used to synchronize the sale order "
+        "date.")
+    product_qty_field = fields.Selection(
+        selection=[
+            ('immediately_usable_qty', 'Immediately usable qty'),
```

Comment:
immediately_usable_qty it seems it was renamed to qty_available_not_res in stock_available_unreserved module